### PR TITLE
init i18n and lang connexion with storyblok

### LINKED ISSUE #66: [storyblok] create internationalisation connexion
we can use nuxt-18n locally.

or create something custom.

Store local locally in localstorage and add store logic.

From here switch local and when we call sb we should add the local we want

### PR DESCRIPTION
**Pull request by f3ltron**
close #66

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -57,10 +57,11 @@
 export default {
   name: 'HomePage',
   asyncData({ app, error }) {
+    const lang = app.store.state.i18n.locale
```
Comment (fuentesloic) [inline pages/index.vue]: Could be `cons { locale } = app.store.state.i18n` 
And then in the `get` we could use `starts_with: `${locale}/events/`

File: components/navbar/navbar-left.vue [hunk 2]
```diff
@@ -1,33 +1,44 @@
...
 <script>
+import { mapGetters } from 'vuex'
+
 export default {
-  name: 'NavBarLeft'
+  name: 'NavBarLeft',
+  computed: {
+    ...mapGetters(['getNav']),
+    availableLocales() {
+      return this.$i18n.locales.filter((i) => i.code !== this.$i18n.locale)
```
Comment (fuentesloic) [inline components/navbar/navbar-left.vue]: NIT but I prefer be obvious with `i` like
``` javascript
return this.$i18n.locales.filter((something) => something.code !== this.$i18n.locale)
```
But maybe it's just me 😸

File: components/navbar/navbar-left.vue [hunk 3]
```diff
@@ -1,33 +1,44 @@
...
-    <nuxt-link class="mb-2 p-1 font-semibold" to="/">
-      <span class="pb-1">Home</span>
-    </nuxt-link>
-    <nuxt-link class="mb-2 p-1 font-semibold" to="/#">
-      <span class="pb-1">Sponsoring us</span>
-    </nuxt-link>
-    <nuxt-link class="mb-2 p-1 font-semibold" to="/#">
-      <span class="pb-1">Gallery</span>
+    <nuxt-link
+      v-for="n in getNav"
```
Comment (fuentesloic) [inline components/navbar/navbar-left.vue]: `n` means `nav`? usually I prefer obvious naming, my 2cts ^^

File: components/navbar/navbar-left.vue (hunk 2 above)
Comment (f3ltron) [reply components/navbar/navbar-left.vue]: i agree

File: components/navbar/navbar-left.vue (hunk 3 above)
Comment (f3ltron) [reply components/navbar/navbar-left.vue]: agree

**Comment by f3ltron** [comment]
the mr is not fully finish i will finish it today later

File: pages/index.vue (hunk 1 above)
Comment (f3ltron) [reply pages/index.vue]: for this its the same

**Pull request merged**